docs/demo.md

```diff
@@ -1,73 +1,325 @@
 ---
 id: demo
-title: components demo
+title: Components demo
 sidebar_label: Components Demo
 ---
 
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-<div className="snippet-container">
-  <Tabs
-    defaultValue="js"
-    values={[
-      { label: 'Javascript', value: 'js', },
-      { label: 'Python', value: 'py', },
-      { label: 'Java', value: 'java', },
-    ]
-  }>
-  <TabItem value="js">
-
-  ```jsx
-  import React, { useState } from "react";
-   
-  function Example() {
-    const [count, setCount] = useState(0);
-     
-    return (
-      <div>
-        <p>You clicked {count} times</p>
-        <button onClick={() => setCount(count + 1)}>
-          Click me
-        </button>
-      </div>
-    );
+<br />
+
+## Matches comps
+
+<MultiStepSnippet
```

Comment:
I can't find the PR for this but please refactor the `MultiStepSnippet` component to use `children` instead of passing in `values`.